server/core/src/main/java/org/jolokia/server/core/service/serializer/SerializeOptions.java

```diff
@@ -47,17 +47,21 @@ public final class SerializeOptions {
     // Maximum number of objects to return
     private final int maxObjects;
 
+    // How to serialize long values: "none" or "string"
```

Comment:
Hmm, I'm just wondering if it should be `"number" or "string"`... `"none"` may be confusing. But it's not a big deal.

Reply:
You are right. `number` should be better. I'll update it. Thanks!